js/index.js

```diff
@@ -0,0 +1,25 @@
+const footerChild = document.createElement('footer');
+document.body.appendChild(footerChild);
+
+let today = new Date();
+let thisYear = today.getFullYear();
+
+const footer = document.querySelector('footer');
+const copyright = document.createElement('p');
+copyright.innerHTML = `Joel DeLeon ${thisYear}`;
+document.body.appendChild(copyright);
```

Comment:
You did great creating the copyright and adding it to your page!
Still, you were supposed to add it to the footer, not directly to the body. You created a footer and never used it.

```suggestion
const footer = document.createElement('footer');
document.body.appendChild(footer);

let today = new Date();
let thisYear = today.getFullYear();

const copyright = document.createElement('p');
copyright.innerHTML = `Joel DeLeon ${thisYear}`;
footer.appendChild(copyright);
```

Reply:
Thank you edited it and pushed the changes!